assets/locales/en/translation.json

```diff
@@ -155,9 +155,9 @@
   "xCoordinate": "X: {{x}}",
   "yCoordinate": "Y: {{y}}",
   "SauceLabs Data Center": "SauceLabs Data Center",
-  "US": "US",
-  "EU": "EU",
-  "UP": "Appium Inspector will use the new Sauce Labs Unified Platform US endpoint by default. If you want to use the old US endpoint then do the following: Go to the 'Custom server'-tab, add 'ondemand.saucelabs.com' as a remote host, use port '433', the remote path is '/wd/hub' and last but not least enable 'SSL'.",
+  "US-West": "US West",
```

Comment:
it is only enough to update it here. All other languages are synchronized automatically by a script.

Reply:
So, revert all other files?

Reply:
Reset the translations